spanner/go.mod

```diff
@@ -12,5 +12,6 @@ require (
 	google.golang.org/api v0.54.0
 	google.golang.org/genproto v0.0.0-20210825212027-de86158e7fda
 	google.golang.org/grpc v1.40.0
+	google.golang.org/grpc/cmd/protoc-gen-go-grpc v1.1.0 // indirect
```

Comment:
I don't think we should add this dependency. We probably should remove the changes in go.mod and go.sum.

Reply:
I've removed both